PubNub/Core/PubNub+Publish.h

```diff
@@ -829,6 +829,21 @@ self.client = [PubNub clientWithConfiguration:configuration];
          compressed:(BOOL)compressed withMetadata:(nullable NSDictionary<NSString *, id> *)metadata
          completion:(nullable PNPublishCompletionBlock)block;
 
+#pragma mark - Fire (no rep)
+
+/**
+ @since 4.4.0
+ */
+- (void)fire:(nullable id)message toChannel:(NSString *)channel completion:(nullable PNPublishCompletionBlock)block;
```

Comment:
`fire:(nullable id)message` - when there is no other potential sources of message payload (like mobile push payload) there shouldn't be _nullable_ in signature.